env-config/changeEnv.js

```diff
@@ -20,18 +20,6 @@ fs.copyFile(
   }
 );
 
-// destination.txt will be created or overwritten by default.
-fs.copyFile(
-  `env-config/env-variables-${argEnv}.json`,
-  'public/env-variables.json',
-  (err) => {
-    if (err) throw err;
-    console.log(
-      `env-config/env-variables-${argEnv}.json was copied to public/env-variables.json`
-    );
-  }
-);
```

Comment:
I think we still need these lines to switch the variables when we change from dev to qa to prod...